### PR TITLE
Add 'Package info' group button #91

### PR DESCRIPTION
**Pull request by johnmhoran**
* Also added CSS in main.css to slightly reduce buttons' left/right
    padding.

Signed-off-by: John M. Horan <johnmhoran@gmail.com>

### CODE REVIEW

File: assets/css/main.css [hunk 1]
```diff
@@ -366,4 +366,15 @@ div.dataTables_scrollHead th:first-child {
   padding: 14px;
   text-align: center;
   color: #ffffff;
+}
+
+/* Modify the colvis column-visibility and -grouping buttons */
```
Comment (jdaguil) [inline assets/css/main.css]: Could you move these styles to be under https://github.com/nexB/aboutcode-manager/blob/9b9f3d5fd7dbea587b49cbbf8ce633f6354d8560/assets/css/main.css#L236-L243

File: assets/js/aboutCodeDataTables.js [hunk 2]
```diff
@@ -245,14 +245,24 @@ class AboutCodeDataTable {
                         .map((column) => `${column.name}:name`)
                 },
                 {
+                    // Show only package columns
```
Comment (jdaguil) [inline assets/js/aboutCodeDataTables.js]: Could you change the order of the buttons to be in the following order:

1. Column Visibility
2. Show all
3. Hide all
4. Origin info
5. Copyright info
6. License info
7. Package info

**Comment by johnmhoran** [comment]
:+1:  Will tackle this afternoon, @jdaguil .

**Comment by jdaguil** [comment]
LGTM!

**Pull request merged**